test/grpc.js

```diff
@@ -46,7 +46,7 @@ describe('grpc', function() {
     });
   });
 
-  describe.skip('metadataBuilder', function() {
+  describe('metadataBuilder', function() {
```

Comment:
Doesn't this mean that the `it('builds metadata'` test is now being run? Is that okay?